.github/workflows/ci-action-ros.yaml

```diff
@@ -21,27 +22,25 @@ jobs:
             submodules: recursive
       - name: Dependencies
         run: |
-            sudo apt install -y libeigen3-dev ros-noetic-cv-bridge python3-pip
-            sudo pip install --upgrade pip
-            sudo python3 -m pip install catkin_tools empy catkin_pkg
+            sudo apt install -y python3-colcon-common-extensions python3-pip ros-jazzy-cv-bridge libeigen3-dev
       - name: Install Ouroboros
-        run: pwd && ls && cd src/ouroboros_repo && pwd && pip install ./ouroboros
+        run: pwd && ls && cd src/ouroboros_repo && pwd && python3 -m pip install --break-system-packages ./ouroboros
       - name: Install ROS packages with rosdep
         shell: bash
         run: |
-          source /opt/ros/noetic/setup.bash
+          source /opt/ros/jazzy/setup.bash
           rosdep update
           rosdep install --from-paths src --ignore-src -r -s  # do a dry-run first
           rosdep install --from-paths src --ignore-src -r -y
-      - name: catkin build
+      - name: colcon build
         shell: bash
         run: |
-          source /opt/ros/noetic/setup.bash
-          catkin build -s
+          source /opt/ros/jazzy/setup.bash
+          colcon build --continue-on-error
```

Comment:
Are there build errors expected? Ideally CI would fail on colcon not building the workspace